controllers/servers/host_definer/watcher/watcher_helper.py

```diff
@@ -0,0 +1,469 @@
+import base64
+import datetime
+import os
+from kubernetes import client, config, dynamic
+from kubernetes.client import api_client
+from kubernetes.client.rest import ApiException
+
+from controllers.common.csi_logger import get_stdout_logger
+from controllers.servers.host_definer.common import settings
+from controllers.servers.host_definer.common.types import DefineHostRequest
+from controllers.servers.host_definer.storage_manager.host import StorageHostManager
+import controllers.servers.host_definer.watcher.exceptions as exceptions
+
+SECRET_IDS = {}
+NODES = {}
+logger = get_stdout_logger()
+
+
+class WatcherHelper:
```

Comment:
I think `watcher` is a better name since every watcher is inheriting it and not "using" it